accession/admin.py

```diff
@@ -1,68 +1,131 @@
 from django.contrib import admin
 
+from accession import models
+
+
 class AccessionAdmin(admin.ModelAdmin):
-    list_display = ['accession_number', 'donor', 'date_received', 'has_date_paperwork_sent', 'has_date_paperwork_returned', 'description', 'accession_note']
-    search_fields = ['accession_number', 'description', 'accession_note', 'donor__first_name', 'donor__last_name',]
+    list_display = ['accession_number', 'donor', 'date_received',
+                    'has_date_paperwork_sent', 'has_date_paperwork_returned',
+                    'description', 'accession_note']
+    search_fields = ['accession_number', 'description', 'accession_note',
+                     'donor__first_name', 'donor__last_name']
     date_hierarchy = 'date_received'
 
+
 class DonorAdmin(admin.ModelAdmin):
-    list_display = ['full_name', 'phone_number_1', 'comments',]
+    list_display = ['full_name', 'phone_number_1', 'comments']
     search_fields = ['first_name', 'last_name', 'email_address', 'comments']
 
     fieldsets = [
         ('Contact Info', {
-            'fields': ('salutation', 'first_name', 'middle_name', 'last_name', 'organization_name','donor_type','gender'),
+            'fields': ('salutation', 'first_name', 'middle_name', 'last_name',
+                       'organization_name', 'donor_type', 'gender')
         }),
         ('Address', {
-            'fields': ('address_1', 'address_2', 'city', 'state', 'postal_code', 'country'),
+            'fields': ('address_1', 'address_2', 'city', 'state',
+                       'postal_code', 'country')
         }),
         ('Contact Info', {
-            'fields': ('phone_number_1', 'phone_number_2', 'email_address'),
+            'fields': ('phone_number_1', 'phone_number_2', 'email_address')
         }),
         ('Comments', {
-            'fields': ('comments',),
+            'fields': ('comments', )
         })
     ]
 
+
 class ObjectAdmin(admin.ModelAdmin):
-    list_display = ['object_number', 'object_description', 'showLabel', 'showLocation', 'showDesigner']
-    search_fields = ['object_number','object_description', 'remarks', 'public_notes', 'label__label']
+    list_display = ['object_number', 'object_description', 'showLabel',
+                    'showLocation', 'showDesigner']
+    search_fields = ['object_number', 'object_description', 'remarks',
+                     'public_notes', 'label__label']
+
 
 class CityAdmin(admin.ModelAdmin):
-    search_fields = ['city']
+    search_fields = ['city', ]
```

Comment:
As discussed, we can actually just remove the comma from the single element lists.